scripts/create-unity-package.sh

```diff
@@ -10,17 +10,28 @@ done
 PROJECT="$PWD/Subtester"
 PACKAGE="$PWD/Purchases.unitypackage"
 PACKAGE_UNITY_IAP="$PWD/Purchases-UnityIAP.unitypackage"
+
+# To export a .unitypackage, we need to have the SDK files inside the Subtester project
+# Additionally, we need to remove the UPM dependency from the manifest to export it,
+# otherwise it will cause issues due to duplicated symbols.
+SYMBOLIC_LINK_PATH="$PROJECT/Assets/RevenueCat"
+MANIFEST_JSON_PATH="$PROJECT/Packages/manifest.json"
+ln -s "$PWD/RevenueCat" "$PROJECT/Assets/"
```

Comment:
So this is creating a link between the `RevenueCat` folder in the root and `Subtester/Assets`. Why?

Reply:
Yes. If we try to export a `.unitypackage` from Subtester it won't export the RevenueCat SDK files even though they are part of the project using a local package. The only way to export a `unitypackage` with these assets is to have these files inside the `Assets` folder of the project. This is only done as part of the process to create the `unitypackage`.

Reply:
definitely worth commenting thoroughly. 
Furthermore... this script just keeps getting more and more complicated over time. We've been putting it off for a while, but it feels like we might have already crossed the threshold where it should be moved to ruby for clarity and maintainability. 

We can do that as a separate task, but it would definitely be easier with this fresh in our memories than when we need another maintainer needs to actually change anything here.

Reply:
+1 for migrating it as a followup

Reply:
Absolutely, I think moving to fastlane makes a lot of sense.

Reply:
Created Jira task for it here: https://revenuecats.atlassian.net/browse/CSDK-601